tools/dgeni/processors/categorizer.js

```diff
@@ -1,3 +1,11 @@
+const DIRECTIVE_SELECTOR_BLACKLIST = [
+  '[portal]',
+  '[portalHost]',
+  'textarea[md-autosize]',
+  '[overlay-origin]',
+  '[connected-overlay]',
```

Comment:
You can make this a `Set` instead of an array:
```ts
const SELECTOR_BLACKLIST = new Set([
  '[portal]',
  '[portalHost]',
  'textarea[md-autosize]',
  '[overlay-origin]',
  '[connected-overlay]',
]);
```
And then use `SELECTOR_BLACKLIST.has(...)` instead of `includes`